### PR TITLE
NFDIV-2024 : welsh notification changes

### PR DESCRIPTION
**Pull request by sunilnatikar**
### JIRA link ###
https://tools.hmcts.net/jira/browse/NFDIV-2024

### Change description ###
Added welsh notification to app 1 for joint case when app2 approved - happy path scenario

**New template added:**
https://www.notifications.service.gov.uk/services/56f68e5c-b51f-4bdf-a0ac-d70af335e1d1/templates/7d8db5c5-4b9c-4221-8fed-8c542d45a6af

**Sample email screen shots:**
**- Respondent responded:**
Screenshot 2022-06-08 at 13 25 20

**- Reminder Cron**
Screenshot 2022-06-08 at 14 10 10

**Before merging a pull request make sure that:**

- [x] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [x] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/divorce/citizen/notification/Applicant2ApprovedNotificationTest.java [hunk 1]
```diff
@@ -237,5 +239,26 @@ void shouldSendEmailToApplicant1SolicitorWithDissolutionContent() {
...
+        data.getApplication().getApplicant2HelpWithFees().setNeedHelp(YesOrNo.YES);
+        data.setDueDate(LOCAL_DATE);
+        data.getApplicant1().setLanguagePreferenceWelsh(YesOrNo.YES);
+
+        notification.sendToApplicant1(data, 1234567890123456L);
+
+        verify(notificationService).sendEmail(
+            eq(TEST_USER_EMAIL),
+            eq(JOINT_APPLICANT1_APPLICANT2_APPROVED),
+            anyMap(),
```
Comment (nitinprabhu) [inline src/test/java/uk/gov/hmcts/divorce/citizen/notification/Applicant2ApprovedNotificationTest.java:257]: for welsh I would also add check something like https://github.com/hmcts/nfdiv-case-api/pull/2003/files#diff-6628fc0f1cc3ddd9bf790381b31bd2a9b892cdd24889fe4feb503a3e08b3720cR137

File: src/test/java/uk/gov/hmcts/divorce/citizen/notification/JointApplicationApprovedReminderTest.java [hunk 2]
```diff
@@ -144,4 +145,33 @@ void shouldSendReminderEmailToApplicant1WithDissolutionAndNoPaymentContent() {
...
+        when(commonContent.mainTemplateVars(data, 1234567890123456L, data.getApplicant1(), data.getApplicant2()))
+            .thenReturn(getMainTemplateVars());
+
+        jointApplicationApprovedReminder.sendToApplicant1(data, 1234567890123456L);
+
+        verify(notificationService).sendEmail(
+            eq(TEST_USER_EMAIL),
+            eq(JOINT_APPLICATION_APPROVED_APPLICANT1_REMINDER),
+            argThat(allOf(
+                hasEntry(IS_REMINDER, YES),
```
Comment (nitinprabhu) [inline src/test/java/uk/gov/hmcts/divorce/citizen/notification/JointApplicationApprovedReminderTest.java:169]: same here https://github.com/hmcts/nfdiv-case-api/pull/2003/files#diff-6628fc0f1cc3ddd9bf790381b31bd2a9b892cdd24889fe4feb503a3e08b3720cR137

File: src/test/java/uk/gov/hmcts/divorce/citizen/notification/JointApplicationApprovedReminderTest.java [hunk 3]
```diff
@@ -144,4 +145,33 @@ void shouldSendReminderEmailToApplicant1WithDissolutionAndNoPaymentContent() {
...
+                hasEntry(PAYS_FEES, NO)
+            )),
```
Comment (mapingo) [inline src/test/java/uk/gov/hmcts/divorce/citizen/notification/JointApplicationApprovedReminderTest.java:171]: Can the Welsh content be verified?
hasEntry(PARTNER, "gŵr")

File: src/test/java/uk/gov/hmcts/divorce/citizen/notification/Applicant2ApprovedNotificationTest.java (hunk 1 above)
Comment (sunilnatikar) [reply src/test/java/uk/gov/hmcts/divorce/citizen/notification/Applicant2ApprovedNotificationTest.java]: done

**Review by mapingo — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/divorce/citizen/notification/JointApplicationApprovedReminderTest.java (hunk 3 above)
Comment (sunilnatikar) [reply src/test/java/uk/gov/hmcts/divorce/citizen/notification/JointApplicationApprovedReminderTest.java:175]: done

**Pull request merged**